front/lib/api/assistant/messages.ts

```diff
@@ -759,7 +759,7 @@ export async function softDeleteUserMessage(
     return new Err(new ConversationError("message_not_found"));
   }
 
-  if (message.userMessage.userId !== user.id) {
+  if (!auth.isAdmin() && message.userMessage.userId !== user.id) {
```

Comment:
shouldn't it be `||`?

I'm admin, you are not

You are deleting your message:
* auth.isAdmin() => false
* message.userMessage.userId !== user.id => false
* !false && false => false
* we don't enter the `if`